doc/source/ray-air/examples/analyze_tuning_results.ipynb

```diff
@@ -0,0 +1,918 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c62191af",
+   "metadata": {},
+   "source": [
+    "# Analyzing results from hyperparameter tuning\n",
+    "In this example we'll run a hyperparameter tuning experiment and analyze the results.\n",
+    "\n",
+    "What we'll cover:\n",
+    "- How to load data from an Sklearn example dataset\n",
+    "- How to initialize an XGBoost trainer\n",
+    "- How to define a search space for regular XGBoost parameters and for data preprocessors\n",
+    "- How to fetch the best obtained result from the tuning run\n",
+    "- How to fetch a dataframe to do further analysis on the results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "41abda7b",
+   "metadata": {},
+   "source": [
+    "First we start with some imports. We'll use the [Covertype dataset](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_covtype.html#sklearn-datasets-fetch-covtype) provided from sklearn to train a multiclass classification task using XGBoost."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "d45b4f69",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from sklearn.datasets import fetch_covtype\n",
+    "\n",
+    "import ray\n",
+    "from ray import tune\n",
+    "from ray.ml import RunConfig\n",
+    "from ray.ml.preprocessors import StandardScaler\n",
+    "from ray.ml.train.integrations.xgboost import XGBoostTrainer\n",
+    "from ray.tune.tune_config import TuneConfig\n",
+    "from ray.tune.tuner import Tuner"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a93b242c",
+   "metadata": {},
+   "source": [
+    "We'll define a utility function to create a Ray Dataset from the Sklearn dataset. We expect the target column to be in the dataframe, so we'll add it to the dataframe manually.\n",
+    "We don't do any test/validation splitting here - pelase check out our other examples for this."
```

Comment:
```suggestion
    "We don't do any test/validation splitting here - please check out our other examples for this."
```

Reply:
Which examples?

Reply:
Happy to link once they are actually integrated in the docs. This is a bit of a chicken egg problem

Reply:
I just removed the comment for now, it doesn't provide anything in the current example